spanner/spanner_snippets/spanner/integration_test.go

```diff
@@ -45,6 +45,7 @@ type sampleFuncWithContext func(ctx context.Context, w io.Writer, dbName string)
 type instanceSampleFunc func(w io.Writer, projectID, instanceID string) error
 type backupSampleFunc func(ctx context.Context, w io.Writer, dbName, backupID string) error
 type createBackupSampleFunc func(ctx context.Context, w io.Writer, dbName, backupID string, versionTime time.Time) error
+type sampleFuncWithProjectID func(w io.Writer, dbName, projectID string) error
```

Comment:
Can we reuse the existing `sampleFunc` and extract `projectID` from `dbName` in the samples? If not, can we change the order of the parameters to `projectID, dbName string`? 

I still think extracting `projectID` from `dbName` might be better, like: 

https://github.com/GoogleCloudPlatform/golang-samples/blob/296e1434dc002ab65f118c40dac5580dc002d451/spanner/spanner_snippets/spanner/integration_test.go#L713-L718

But of course, there are some code duplications. 

Or we should have a 

```go
type databaseSampleFunc func(w io.Writer, projectID, instanceID, databaseID string) error
```

Reply:
Makes sense, changed to parse db name to get project IDs.